app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewsController < ApplicationController
+  def show
+    @card = Card.random_for_review.first
+  end
+
+  def create
+    @card = Card.find(params[:card_id])
+
+    if @card.check_user_answer(params[:answer])
+      flash[:success] = 'Верно!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.